templates/pkg/resource/manager.go.tpl

```diff
@@ -177,6 +177,7 @@ func (rm *resourceManager) onError(
 	r *resource,
 	err error,
 ) (acktypes.AWSResource, error) {
+	rm.log.Info("Error occurred.", "Error: ", err.Error())
```

Comment:
remove this please.

Reply:
@kumargauravsharma please remove the above.

Reply:
Removed the changes done to template file.